proto/user-service.proto

```diff
@@ -52,6 +64,20 @@ message UpdateUserResponse {
     User user = 1;
 }
 
+message ValidateTokenResponse {
+    bool valid = 1;
+}
+
+message LoginRequest {
+    string username = 1;
```

Comment:
The token does contains the users username "amongst other things" and we verify the tokens content with the username of the user which the token belongs to. Otherwise we'll have to do that check somewhere else to make sure that the owner of the token is the rightful one. 

Perhaps i've misintepreted the question? 

But the reason why Auth request takes a username aswell is to verify that the token comes from the correct user. Perhaps that check is done somewhere else i'm not aware of, in that case i'll remove the username field.